connector/mqtt/vernemq/v2k-bridge/app/MQTTClient.js

```diff
@@ -110,6 +107,20 @@ class MQTTClient {
     this.mqttc.reconnect();
   }
 
+  /**
+   * Handles MQTT errors.
+   *
+   * @param {*} error
+   */
+  onError(error) {
+    this.logger.error('An error has occurred in the MQTT connection.');
+    if (error) {
+      this.logger.error(error.stack || error);
+    }
+    this.logger.error('Bailing out!');
+    process.exit(1);
```

Comment:
Shouldn't be better to use `process.kill` as done in K2V?

Reply:
I don't know, because this service does not have the problem that K2V had. I can change it, but it does not seem to be really necessary.

Reply:
Ok! Let's keep this way!